src/ds/logger.h

```diff
@@ -166,8 +175,9 @@ namespace logger
         // Padding on the right to align the rest of the message
         // with lines that contain enclave time offsets
         return fmt::format(
-          "{}        [{:<5}] {:<36} | {}",
+          "{}        {} [{:<5}] {:<36} | {}",
```

Comment:
We should be formatting the `thread_id` to a fixed width so that the log contents remain aligned.